src/widget/Mainframe.cpp

```diff
@@ -520,8 +542,13 @@ void Mainframe::open() {
     }
   }
 
-  QString retValue =
+  QString retValue;
+  if (dir == nullptr) {
```

Comment:
```suggestion
  if (dir.isNull()) {
```
See https://doc.qt.io/qt-6/qstring.html#distinction-between-null-and-empty-strings